README.md

```diff
@@ -183,6 +190,30 @@ dynamic_batching {
 **NOTE:** At this time, the FIL backend supports **only** `TYPE_FP32` for input
 and output. Attempting to use any other type will result in an error.
 
+To enable Shapley value outputs, modify the `output` section of the above example
+to look like:
+
+```
+output [
+ {
+    name: "output__0"
+    data_type: TYPE_FP32
+    dims: [ 2 ]
+  },
+ {
+    name: "treeshap_output"
+    data_type: TYPE_FP32
+    dims: [ 501 ]
+  }
+]
+```
+
+**NOTE:** The dimensions of Shapley value outputs for a multi-class problem
+are of the form `[ n_classes, (n_features + 1) ]`, while for a binary-class problem
+they are of the form `[ n_features + 1 ]`. The additional column in the output stores 
+the bias term. At this moment, Shapley value support is only enabled for configurations
+where `instance_group [{ kind: KIND_GPU }]`.
```

Comment:
A more human readable sentence that says they are only available for GPU would be good